source/help/settings/theme-colors.rst

```diff
@@ -0,0 +1,138 @@
+.. _theme-colors:
+
+Theme Colors
+============
+
+The colors of the Mattermost user interface are customizable in **Account Settings > Display > Theme**. You can import your theme colors from Slack, customize the colors yourself, or choose from four standard themes designed by the Mattermost team.
+
+In Enterprise Edition, if you belong to multiple teams, you can choose to apply any changes across teams by selecting "Apply New Theme to All Teams" before saving. If this is not selected, any changes are only applied to the current team.
```

Comment:
It's the first `by default it's applied to all teams and you can choose to apply to the current team only`, both in TE and EE.

Then in EE you have the additional option of applying a theme on for the current team only.